plugins/push/frontend/public/javascripts/countly.views.js

```diff
@@ -1173,28 +1176,28 @@
                 }
             },
             shouldShowDuplicateUserCommand: function() {
-                return true;
+                return this.canUserCreate;
```

Comment:
I included the push notificaiton plugin permissions on this PR. I thought it should be straightforward.